api/prove.js

```diff
@@ -132,15 +136,43 @@ switch (options.inputgen || options.test || options.prove) {
 
   // Prove mode
   case options.prove === true:
-    let result = await zkgapi.prove(
-      wasmUnit8Array,
+    const feeInWei = ethers.utils.parseEther("0.005");
+    const provider = new ethers.providers.JsonRpcProvider(
+      config.DispatcherProviderUrl
+    );
+    const signer = new ethers.Wallet(config.UserPrivateKey, provider);
+
+    let dispatcherContract = new ethers.Contract(
+      config.DispatcherContract,
+      TdABI,
+      provider
+    ).connect(signer);
+
+    const md5 = ZkWasmUtil.convertToMd5(wasmUnit8Array).toUpperCase();
+    const tx = await dispatcherContract.prove(
+      md5,
       privateInputStr,
       publicInputStr,
-      config.ZkwasmProviderUrl,
-      config.UserPrivateKey,
-      enableLog,
+      {
+        value: feeInWei,
```

Comment:
we need to add a warning about "You are going to publish a Setup/Prove/Deploy request to the Sepolia testnet, which would require {} SepoliaETH. \n Hit Enter to confirm." before sending out the tx.

Reply:
@fewwwww please review the notice msg